Source/Core/DolphinQt2/Settings/AudioPane.cpp

```diff
@@ -0,0 +1,228 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Settings/AudioPane.h"
+
+#include <QCheckBox>
+#include <QComboBox>
+#include <QFormLayout>
+#include <QGridLayout>
+#include <QGroupBox>
+#include <QLabel>
+#include <QRadioButton>
+#include <QSlider>
+#include <QSpinBox>
+#include <QVBoxLayout>
+
+#include "AudioCommon/AudioCommon.h"
+#include "DolphinQt2/Config/SettingsWindow.h"
+#include "DolphinQt2/Settings.h"
+
+AudioPane::AudioPane()
+{
+  CreateWidgets();
+  LoadSettings();
+  ConnectWidgets();
+
+  connect(&Settings::Instance(), &Settings::VolumeChanged, this, &AudioPane::OnVolumeChanged);
+}
+
+void AudioPane::CreateWidgets()
+{
+  auto* dsp_box = new QGroupBox(tr("DSP Emulator Engine"));
+  auto* dsp_layout = new QVBoxLayout;
+  dsp_box->setLayout(dsp_layout);
+  m_dsp_hle = new QRadioButton(tr("DSP HLE emulation (fast)"));
+  m_dsp_lle = new QRadioButton(tr("DSP LLE recompiler"));
+  m_dsp_interpreter = new QRadioButton(tr("DSP LLE interpreter (slow)"));
+
+  dsp_layout->addWidget(m_dsp_hle);
+  dsp_layout->addWidget(m_dsp_lle);
+  dsp_layout->addWidget(m_dsp_interpreter);
+
+  auto* volume_box = new QGroupBox(tr("Volume"));
+  auto* volume_layout = new QGridLayout;
+  m_volume_slider = new QSlider;
+  m_volume_indicator = new QLabel();
+
+  volume_box->setLayout(volume_layout);
+
+  m_volume_slider->setMinimum(0);
+  m_volume_slider->setMaximum(100);
+
+  m_volume_indicator->setAlignment(Qt::AlignVCenter | Qt::AlignHCenter);
+
+  volume_layout->addWidget(m_volume_slider, 1, 2);
+  volume_layout->addWidget(m_volume_indicator, 2, 1, 1, 3);
+
+  auto* backend_box = new QGroupBox(tr("Backend Settings"));
+  auto* backend_layout = new QFormLayout;
+  backend_box->setLayout(backend_layout);
+  m_backend_label = new QLabel(tr("Audio Backend"));
```

Comment:
Willing to change it, but 
> Arguably, the controller settings don't use colons because it's they're so complicated already, I guess?
greatly confuses me.